src/components/Provider/context.client.ts

```diff
@@ -26,7 +31,12 @@ export const NostoContext = createContext<NostoContextType>({
   pageType: "",
   responseMode: "HTML",
   clientScriptLoaded: false,
-  useRenderCampaigns: () => undefined,
+  useRenderCampaigns: () => {
```

Comment:
@sid2364 is this change problematic?
the previous logic was problematic from TS type checking point of view